src/components/React/ColorMappingController.js

```diff
@@ -1,73 +1,31 @@
 import React, { Component } from "react";
 import ReactModal from "react-modal";
+// import { DataTable } from "primereact/datatable";
+// import { Column } from "primereact/column";
+import "primereact/resources/themes/nova/theme.css";
+import "primereact/resources/primereact.min.css";
```

Comment:
is the primereact really needed?

Reply:
It's all for the Sidebar component. react-bootstrap doesn't come with one out of the box

Reply:
#39